flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -266,11 +305,16 @@ public RocksDBKeyedStateBackend(
 		RocksIterator iterator = db.newIterator(columnInfo.f0);
 		iterator.seekToFirst();
 
-		Iterable<K> iterable = () -> new RocksIteratorWrapper<>(iterator, state, keySerializer, keyGroupPrefixBytes);
+		Iterable<K> iterable = () -> new RocksIteratorToJavaIteratorAdapter<>(iterator, state, keySerializer, keyGroupPrefixBytes);
 		Stream<K> targetStream = StreamSupport.stream(iterable.spliterator(), false);
```

Comment:
This method seems to be problematic, it doesn't use the `namespace` to filter data ...

Reply:
Yes, that seems a little odd, but I suggest to make this a separate issue because this PR has not introduced or changed this, except for that the class was renamed.

Reply:
Aha, I would like to open a issue with a PR for this if you don't mind?

Reply:
Fine with me, but since this seems to have not caused any trouble so far, you might first want to check if the namespace parameter is actually required. Maybe it can just be removed as parameter because the using code wants to iterate all keys in all namespace.

Reply:
Ok, I will check whether the namespace parameter is actually required by check the other state backend (HeapKeyedStateBackend), I think their implementation should be consistent.

Reply:
Checked. In `HeapKeyedStateBackend` the implementation is 
```
	@Override
	public <N> Stream<K> getKeys(String state, N namespace) {
		if (!stateTables.containsKey(state)) {
			return Stream.empty();
		}
		StateTable<K, N, ?> table = (StateTable<K, N, ?>) stateTables.get(state);
		return table.getKeys(namespace);
	}
```
So I think `namespace` is useful ...

Reply:
👍

Reply:
Well, maybe, except if this is also only used with void-namespace.

Reply:
After tracing the invoked path, I found this seem to be a new function. And there is a  block bug I found that relate to it ... [5513](https://github.com/apache/flink/pull/5513)